src/modules/vtol_att_control/vtol_att_control_main.cpp

```diff
@@ -156,18 +156,29 @@ VtolAttitudeControl::vehicle_cmd_poll()
 void
 VtolAttitudeControl::handle_command()
 {
-	// update transition command if necessary
 	if (_vehicle_cmd.command == vehicle_command_s::VEHICLE_CMD_DO_VTOL_TRANSITION) {
-		_transition_command = int(_vehicle_cmd.param1 + 0.5f);
 
-		// Report that we have received the command no matter what we actually do with it.
-		// This might not be optimal but is better than no response at all.
+		vehicle_status_s vehicle_status = {};
+		_vehicle_status_sub.update(&vehicle_status);
```

Comment:
```suggestion
		_vehicle_status_sub.copy(&vehicle_status);
```

Reply:
The update method will only copy the data if the data has actually updated since the last check.